lib/Importer/Caffe2.cpp

```diff
@@ -134,24 +134,22 @@ void caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     size_t depth = wtag.dims()[0];
 
     // Construct the Filter field.
-    Variable *filter =
-        G_.getParent()->createVariable(&wtag.getType(), "conv.filter");
-    filter->getPayload().copyFrom(&wtag);
+    auto *filter = createNode("conv.filter", wtag);
```

Comment:
We can't tell from reading this line that we are creating a variable. We need a better name for the API.

Reply:
Definitely go with createVariable for this PR then.
I'll post an update with just this part of the change and we'll decide on the broader renaming/API clean-up if we want to squash it into this.